### PR TITLE
Update typings config and examples

### PR DESCRIPTION
**Pull request by edgarmueller**
Changes regarding typings 1.x.x

### CODE REVIEW

File: src/components/renderers/controls/array/array-directive.ts [hunk 1]
```diff
@@ -63,9 +69,11 @@ class ArrayController extends AbstractControl {
         let resolvedSubSchema = PathResolver.resolveSchema(
             this.schema, this.schemaPath) as SchemaArray;
         let items = resolvedSubSchema.items;
+        console.log('items are ' + JSON.stringify(items));
```
Comment (eneufeld) [inline src/components/renderers/controls/array/array-directive.ts]: did you forget to remove this?

File: src/components/renderers/controls/array/array-directive.ts [hunk 2]
```diff
@@ -63,9 +69,11 @@ class ArrayController extends AbstractControl {
...
         this.arraySchema = items;
         this.properties = _.keys(items['properties']);
-        this.arrayUiSchema = uiGenerator.generateDefaultUISchema(items);
+        this.arrayUiSchema = uiGenerator.generateDefaultUISchema(items, 'HorizontalLayout');
+        console.log('array ui schema is ' + JSON.stringify(this.arrayUiSchema));
```
Comment (eneufeld) [inline src/components/renderers/controls/array/array-directive.ts]: and this?

File: src/components/renderers/controls/array/array-directive.ts [hunk 3]
```diff
@@ -21,18 +19,23 @@ class ArrayReadOnlyDirective implements ng.IDirective {
...
-class ArrayDirective implements ng.IDirective {
-    restrict = 'E';
-    template = `
+const arrayTemplate = `
     <jsonforms-layout>
         <fieldset ng-disabled="vm.uiSchema.readOnly">
           <legend>{{vm.label}}</legend>
           <div>
-            <div ng-repeat="d in vm.resolvedData[vm.fragment]" class="well">
+            <div ng-repeat="d in vm.resolvedData" ng-if="vm.fragment === undefined" class="well well-sm">
```
Comment (eneufeld) [inline src/components/renderers/controls/array/array-directive.ts]: are those classes bootstrap specific?

**Comment by eneufeld** [comment]
looks good

**Comment by edgarmueller** [comment]
Geez, the build keeps on failing. I'm merging this now, but I've opened [a new issue](https://github.com/eclipsesource/jsonforms/issues/360) to review our build

**Pull request merged**